webapp.py

```diff
@@ -114,7 +114,9 @@ def attendee(id):
         p = db.Person.get(id = id)
         if flask.request.args.get('auth') == p.auth():
             return flask.render_template('attendee.html',
-                attendee = p, products = db.Product.select())
```

Comment:
This all looks sensible, but would you mind breaking the code changes out from the template changes into a separate pull request? Thanks.

Reply:
template change it separated to #12